specification/codesigning/Microsoft.CodeSigning/tspconfig.yaml

```diff
@@ -0,0 +1,10 @@
+parameters:
+  service-directory-name:
+    default: "codesigning"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
```

Comment:
I assume you want to generate SDK's and if so, you will want add the other sdk configurations. See our sample at https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml.

Reply:
I used the sample for now to add those emitters. Thanks